src/drc/api/serializers.py

```diff
@@ -4,7 +4,9 @@
 import uuid
 
 from django.conf import settings
+from django.db import transaction
 from django.utils.encoding import force_text
```

Comment:
Couple of unused imports and linter errors in general, can you check your linter setup?